### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR includes 

- Navigation component 
- Pages are updated with wrapper div
- Added new Home page

### CODE REVIEW

**Review by AnuragSahu11 — APPROVED** [review]
Using TS 🔥 and tailwind, good folder structure ready to merge

File: src/components/Aside/Aside.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            </Link>
+          </li>
+          <li className="flex items-center text-xl my-5 py-2 pl-5 whitespace-nowrap hover:bg-hoverClr hover:rounded-r-full">
+            <span>
+              <AiOutlineEdit className="mr-2" />
+            </span>
+            Edit labels
+          </li>
+          <li className="pl-5 hover:bg-hoverClr hover:rounded-r-full">
+            <Link
```
Comment (shraddha15-creator) [inline src/components/Aside/Aside.tsx]: You can use NavLink instead of Link.

**Review by shraddha15-creator — APPROVED** [review]
Good Folder structure.
UI looks nice and clean

File: src/components/Aside/Aside.tsx (hunk 1 above)
Comment (junaidshaikh-js) [reply src/components/Aside/Aside.tsx]: Thank you for the suggestion, Shraddha. 

I will make changes in later PRs

**Pull request merged**